app/views/projects/workflow_executions/index.html.erb

```diff
@@ -26,8 +26,9 @@
   <% end %>
 <% end %>
 <div class="flow-root">
+  <div class="flex mb-2">
   <% if allowed_to?(:export_data?, @project.namespace) %>
-    <div class="float-left inline-flex space-x-2 mb-4">
+    <div class="inline-flex grow space-x-2 mb-4">
```

Comment:
Can you remove the `mb-4` from here.  It does not cause an issue for some reason in the UI, but it could because it is adding a margin to the bottom of the buttons only not the search box.  The surrounding div already give a margin of `mb-2` which handles both.  Interestingly, when I inspect the div in chrome it has stripped out the `mb-4` here (and I don't know why).
![alt text](image.png)

Reply:
Removed class and added highlighting in b01a2cbfb120d3b32ad3713be8073819d5016c2a

Reply:
Thanks, that looks good